src/merge-info.js

```diff
@@ -1,43 +0,0 @@
-'use strict';
-
-const moment = require('moment');
-
-module.exports = (npm, gh) => {
-  gh = gh || {};
-  let ownerWebsite = npm.author && npm.author.url;
-  ownerWebsite = ownerWebsite || gh.owner && gh.owner.html_url;
-  ownerWebsite = ownerWebsite || '';
-
-  return {
-    description: cleanupDescription(npm.description || gh.description || ''),
-    downloads: npm.downloads,
-    name: npm.name.replace(/^generator-/, '').trim(),
-    official: ownerWebsite && ownerWebsite === 'https://github.com/yeoman',
-    owner: {
-      name: npm.author && npm.author.name || gh.owner && gh.owner.login || '',
-      site: ownerWebsite
-    },
-    site: npm.website || gh.html_url || '',
-    stars: gh.stargazers_count || 0,
-    timeSince: moment(npm.updated).fromNow(),
-    updated: npm.updated
-  };
-};
-
-function cleanupDescription(str) {
-  if (!str) {
-    return '';
-  }
-
-  str = str.trim()
```

Comment:
this part should be left in unless npm.io somehow already follows these rules, which I doubt